src/api/habits/adapters/json.ts

```diff
@@ -26,7 +30,75 @@ const jsonHabitServiceFactory = (): HabitService => {
         }))
       );
 
-  return { addHabit, getHabits };
+  const addEntry = (
+    goals: GoalWithHabitHistory[],
+    habitId: string,
```

Comment:
Is the ID just the name, or something else, btw? If it's something else, it may be better to generate the id as the return value, rather than allow for entry

Reply:
Well, I'm glad you asked that because I noticed an issue. At the moment, the entry key is the habit ID, not an entry ID. I did that temporarily to get the code working but I didn't update it. I'll fix that now.

And to answer the question, the habit IDs are currently just numbers (but stored as strings). I can't remember why I did that but it's okay because it allows us to use a string later (which I've seen APIs like uuid and Firestore do) without going through code/type changes.